### DISCUSSION TITLE
problem with go mod

### DISCUSSION THREAD

**Discussion by fishincat** [juicedata/juicefs#596] [General]
go mod tidy has problem as below:
`github.com/juicedata/juicefs/pkg/object imports
	github.com/ks3sdklib/aws-sdk-go/aws tested by
	github.com/ks3sdklib/aws-sdk-go/aws.test imports
	github.com/ks3sdklib/aws-sdk-go/service/dynamodb: module github.com/ks3sdklib/aws-sdk-go@latest found (v1.0.12), but does not contain package github.com/ks3sdklib/aws-sdk-go/service/dynamodb`

refs: https://github.com/ks3sdklib/aws-sdk-go/issues/12

**Comment by davies** [comment]
@fishincat  We have upgrade ks3sdklib/aws-sdk-go to v1.0.12, this should be solved, please try again.
